IFC4x3/Templates/Partial Templates/Quantities/DocTemplateDefinition.xml

```diff
@@ -7,13 +7,5 @@
 			</Rules>
 		</DocModelRuleAttribute>
 	</Rules>
```

Comment:
Was thinking about this. Does it make sense to completely remove these "Templates" nodes when writing the files? I guess the way it is for now I merge. But these hrefs look okay actually.